src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/dto/CcdBundleDTO.java

```diff
@@ -228,12 +231,16 @@ public void setEnableEmailNotification(CcdBoolean enableEmailNotification) {
     }
 
     @JsonIgnore
-    public void setEnableEmailNotificationAsBoolean(boolean enableEmailNotification) {
-        this.enableEmailNotification = enableEmailNotification ? CcdBoolean.Yes : CcdBoolean.No;
+    public void setEnableEmailNotificationAsBoolean(Boolean enableEmailNotification) {
+        if (enableEmailNotification == null) {
+            this.enableEmailNotification = null;
+        } else {
+            this.enableEmailNotification = enableEmailNotification ? CcdBoolean.Yes : CcdBoolean.No;
+        }
     }
 
     @JsonIgnore
-    public boolean getEnableEmailNotificationAsBoolean() {
-        return enableEmailNotification == CcdBoolean.Yes ? true : false;
+    public Boolean getEnableEmailNotificationAsBoolean() {
+        return enableEmailNotification != null ? enableEmailNotification == CcdBoolean.Yes : null;
```

Comment:
I am wondering what happens if the field is present in the CCD definition file but set to false. With this code it will be null and omitted when sent back to CCD.... is that going to be okay? Or perhaps, if it is set to false we should return false.